frontend/src/components/regions/CardDetailsModal/ReviewsSbonelo.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
+import CardActions from "@material-ui/core/CardActions";
+import IconButton from "@material-ui/core/IconButton";
+import Typography from "@material-ui/core/Typography";
+import CheckCircleIcon from "@material-ui/icons/CheckCircle";
+import Chip from "@material-ui/core/Chip";
+import {
+  createTheme,
+  responsiveFontSizes,
+  MuiThemeProvider,
+} from "@material-ui/core";
+
+import ReviewStatus from "../../widgets/ReviewStatus";
+import ReviewValidationIcons from "../../widgets/ReviewValidationIcons";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: false,
+    transition: "transform 0.15s ease-in-out",
+    "&:hover": { transform: "scale3d(1.05, 1.05, 1)" }
+  },
+}));
+
+export default function Reviews() {
+  const classes = useStyles();
+
+  let theme = createTheme();
+  theme = responsiveFontSizes(theme);
+
+  const review =
```

Comment:
These should come from props. Please dont hard-code values except if you are working in a story

Reply:
This has been modified and not hardcoded